src/database_connect.py

```diff
@@ -1,9 +1,9 @@
 import argparse
 
-from utils import convert_dtypes, data, database, dbconnect, table
+from utils import convert_dtypes, data, database, dbconnect, table, aws_auth, upload_to_s3, download_from_s3
```

Comment:
you don't need to import all the functions, only import the ones that you are using inside the script